core/trino-main/src/main/java/io/trino/execution/scheduler/BinPackingNodeAllocatorService.java

```diff
@@ -209,7 +209,6 @@ synchronized void processPendingAcquires()
     {
         // process EAGER_SPECULATIVE first; it will not cause deadlock even though STANDARD acquires are processed later.
         // STANDARD acquires will ignore EAGER_SPECULATIVE tasks when looking for free resources.
-        // TODO(lukaszos) ADD TEST
```

Comment:
remove in the right commit (currently it's in `Eagerly enumerate splits for eager stages`)